bahmni_purchase/models/product.py

```diff
@@ -4,77 +4,6 @@
 class product_product(models.Model):
     _inherit = 'product.product'
 
-    def get_mrp(self, supplier_id, prod_qty, date=False, uom_id=False):
-        '''Method which will return mrp for an product from Vendor configuration if set,
-            else will return mrp configured in product master'''
-        seller = self._select_seller(partner_id=supplier_id,
-                                     quantity=self.product_qty,
-                                     date=date,
-                                     uom_id=uom_id)
-        if seller:
-            mrp = seller.mrp
-        else:
-            mrp = self.mrp
-        return mrp
 
-    def set_mrp(self, supplier_id, product_qty, mrp, price_unit, date=False, uom_id=False):
-        '''Method which will add vendor configuration if does not exists, 
-            if exists it will set mrp in that.'''
-        seller = self._select_seller(partner_id=supplier_id,
-                                     quantity=self.product_qty,
-                                     date=date,
-                                     uom_id=uom_id)
-        if seller:
-            seller.mrp = mrp
-        else:
-            max_seq = self.env['product.supplierinfo'].search([], order='sequence desc', limit=1)
-            sequence = max_seq + 1
-            self.env['product.supplierinfo'].create({'product_tmpl_id': self.product_tmpl_id.id,
-                                                     'name': supplier_id,
-                                                     'min_qty': product_qty,
-                                                     'price': price_unit,
-                                                     'mrp': mrp,
-                                                     'sequence': sequence})
 
-    def _check_low_stock(self):
```

Comment:
Are all these methods not being used anywhere? Is that why these have been deleted?

Reply:
Yes, It's not used, so We removed.